sourcegraph/jetbrains
# fix/Remote: Make chat work in remote mode

## Test plan

Tested manually:

```
./gradlew :customRunIde -PplatformRuntimeVersion=2024.2 -PplatformRuntimeType=IU -PsplitMode=true
```

1. Sign in to Cody and do some chats.
2. Change themes, check that the theme updates.

Also works in Remote:

<img src="https://github.com/user-attachments/assets/c879fccf-0f0a-4f06-87e4-93cc4051c03f" alt="Screenshot">


## Changelog

- Chat works in remote mode.
